libioc/LaunchableResource.py

```diff
@@ -162,6 +166,17 @@ def rc_conf(self) -> 'libioc.Config.Jail.File.RCConf.ResourceRCConf':
             )
         return self._rc_conf
 
+    @property
+    def periodic_conf(self) -> 'libioc.Config.Jail.File.PeriodicConf.ResourcePeriodicConf':
+        """Return a lazy-loaded instance of the resources PeriodicConf."""
+        if self._periodic_conf is None:
+            import libioc.Config.Jail.File.PeriodicConf
+            self._periodic_conf = libioc.Config.Jail.File.PeriodicConf.ResourcePeriodicConf(
```

Comment:
```
./libioc/LaunchableResource.py:174:80: E501 line too long (92 > 79 characters)
```

Reply:
[yapf](https://github.com/google/yapf) doesn't know how to break this down either.